src/Tribe/Editor/Blocks/Rsvp.php

```diff
@@ -50,10 +50,16 @@ public function default_attributes() {
 	 * @return string
 	 */
 	public function render( $attributes = array() ) {
-		/** @var Tribe__Tickets__Editor__Template $template */
-		$template = tribe( 'tickets.editor.template' );
+		$post_id = get_the_ID();
 
-		$post_id = $template->get( 'post_id', null, false );
+		if ( ! empty( $attributes['post_id'] ) ) {
+			$post_id = $attributes['post_id'];
+		}
+
+		// Prevent the render when the ID of the post has not being set to a correct value
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: e51c882fdcf3557c430a3f20bef3227e</sub>

Reply:
```suggestion
		// Prevent the render when the ID of the post has not being set to a correct value.
```